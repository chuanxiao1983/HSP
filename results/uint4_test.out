SET client_min_messages TO WARNING;
CREATE EXTENSION IF NOT EXISTS hsp;
CREATE EXTENSION IF NOT EXISTS hstore;
select '4294967295'::uint4;
   uint4    
------------
 4294967295
(1 row)

select 4294967295::uint4;
   uint4    
------------
 4294967295
(1 row)

select uint4_hamming(1,4294967295);
 uint4_hamming 
---------------
            31
(1 row)

select hashuint4(1);
  hashuint4  
-------------
 -1905060026
(1 row)

select '4294967295'::uint4 = '4294967295'::uint4;
 ?column? 
----------
 t
(1 row)

