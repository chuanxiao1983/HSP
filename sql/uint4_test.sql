CREATE EXTENSION hsp;
CREATE EXTENSION hstore;

select '4294967295'::uint4;
select 4294967295::uint4;
select uint4_hamming(1,4294967295);
select hashuint4(1);
select '4294967295'::uint4 = '4294967295'::uint4;
