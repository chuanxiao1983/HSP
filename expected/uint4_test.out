   uint4    
------------
 4294967295
(1 row)

   uint4    
------------
 4294967295
(1 row)

 uint4_hamming 
---------------
            31
(1 row)

  hashuint4  
-------------
 -1905060026
(1 row)

 ?column? 
----------
 t
(1 row)

